### PR TITLE
My VA Apply For Benefits visual updates

### PR DESCRIPTION
**Pull request by erikphansen**
## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/appointments/actions/index.js [hunk 1]
```diff
@@ -90,10 +90,12 @@ export function fetchConfirmedFutureAppointments() {
         ccAppointments = MOCK_CC_APPOINTMENTS;
       } else {
         vaAppointments = await apiRequest(
-          `/vaos/v0/appointments?start_date=${startOfToday}&type=va`,
+          `/appointments?start_date=${startOfToday}&type=va`,
+          { apiVersion: 'vaos/v0' },
```
Comment (sanlouise) [inline src/applications/personalization/appointments/actions/index.js:94]: 🙏 Thanks for adding this in!

**Review by sanlouise — APPROVED** [review]
Looks good to me! 👍 🔥

File: src/applications/personalization/dashboard-2/components/CTALink.jsx [hunk 2]
```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 
+import useLastWord from '../useLastWord';
+
 const CTALink = ({ ariaLabel, href, text, onClick, newTab }) => {
   const relProp = newTab ? 'noreferrer noopener' : undefined;
   const targetProp = newTab ? '_blank' : undefined;
 
+  const [lastWord, firstWords] = useLastWord(text);
```
Comment (erikphansen) [inline src/applications/personalization/dashboard-2/components/CTALink.jsx:9]: Definitely open to changing the name of this hook. But it returns an array of two strings:
1. the last word of the passed-in string
2. everything leading up to the last word

File: src/applications/personalization/dashboard-2/components/CTALink.jsx [hunk 3]
```diff
@@ -13,11 +17,14 @@ const CTALink = ({ ariaLabel, href, text, onClick, newTab }) => {
       onClick={onClick || undefined}
       className="vads-u-margin-top--2 vads-u-display--inline-block"
     >
-      {text}
-      <i
-        aria-hidden="true"
-        className="fas fa-xs fa-chevron-right vads-u-margin-x--1"
-      />
+      {`${firstWords} `}
+      <span style={{ whiteSpace: 'nowrap' }}>
```
Comment (erikphansen) [inline src/applications/personalization/dashboard-2/components/CTALink.jsx:21]: We don't have a util class for `nowrap`. But this fix works:

**408px wide**


**409px wide**

Comment (erikphansen) [reply src/applications/personalization/dashboard-2/components/CTALink.jsx:21]: You can play with this locally by changing [this test](https://github.com/department-of-veterans-affairs/vets-website/blob/3b60bb1a209a4bb0b1395bfb301ec498ee882b4f/src/applications/personalization/dashboard-2/tests/e2e/loa3.cypress.spec.js#L86) to `.only` and then mess with the viewport,  `cy.viewport(409, 800)`, on line 87

**Pull request merged**